scrapylib/deltafetch.py

```diff
@@ -36,7 +36,7 @@ class DeltaFetch(object):
 
     """
 
-    def __init__(self, dir, reset=False):
+    def __init__(self, dir, stats=None, reset=False):
```

Comment:
Not sure it could happen in practice, but for those subclassing `DeltaFetch` and using positional arguments, non-`None` stats could trigger a `reset=True` (although passing 3 arguments could break their init before that)

For backward compatibility, can you pass `stats` after `reset=False`?